documentation/bolt_project_directories.md

```diff
@@ -97,7 +113,7 @@ The default paths for all Bolt configuration, code, and data are relative to the
 |[`bolt.yaml`](bolt_configuration_reference.md)|Contains configuration options for Bolt.|
 |`hiera.yaml`|Contains the Hiera config to use for target-specific data when using `apply`.|
 |[`inventory.yaml`](inventory_file_v2.md)|Contains a list of known targets and target specific data.|
-|[`project.yaml`](bolt_configuration_reference.md#project_configuration_options)|Contains configuration for the Bolt project.  The `project.yaml` file contains a whitelist of tasks and plans that you can use to limit the output from the `bolt plan show` and `bolt task show` commands. This is an experimental feature. For more information, see [Bolt projects](./experimental_features.md#bolt-projects).|
+|[`bolt-project.yaml`](bolt_configuration_reference.md#project_configuration_options)|Contains configuration for the Bolt project.  The `bolt-project.yaml` file contains a whitelist of tasks and plans that you can use to limit the output from the `bolt plan show` and `bolt task show` commands. This is an experimental feature. For more information, see [Bolt projects](./experimental_features.md#bolt-projects).|
```

Comment:
I think https://github.com/puppetlabs/bolt/blob/master/documentation/bolt_configuration_reference.md.erb#L87 also needs an update. Might also be a good chance to change that link to point to the experimental features page.